### PR TITLE
Add support for Pygments lexing

### PR DESCRIPTION
**Pull request by pombredanne**
With Pygments we can now collect source symbols, strings and comments.

Reference: https://github.com/nexB/purldb/issues/388

### CODE REVIEW

File: tests/test_symbols_pygments.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/source_inspector for support or download.
```
Comment (keshav-space) [inline tests/test_symbols_pygments.py:7]:
```suggestion
# See https://github.com/nexB/source-inspector for support or download.
```

**Review by keshav-space — APPROVED** [review]
@pombredanne LGTM, few minor nits for your consideration.

**Comment by pombredanne** [comment]
@keshav-space Thanks for the review... All nits applied.

**Comment by keshav-space** [comment]
Merging!

**Pull request merged**